lib/Sounds.ts

```diff
@@ -0,0 +1,23 @@
+let zzfx = (..._: any) => {
+  console.error("zzfx not loaded");
+};
```

Comment:
This blackhole type is to get around SSR since sounds are only available on the client.

I'm okay with the lack of type safety since sound is firmly an enhancement and lack of sound does not impact behaviour.